src/core/scope.ml

```diff
@@ -546,8 +546,8 @@ let scope_rule : sig_state -> p_rule -> pre_rule loc = fun ss r ->
   let pr_vars =
     let fn i =
       let name =
-        try Printf.sprintf "v%i_%s" i (Hashtbl.find lhs_names i)
-        with Not_found -> Printf.sprintf "v%i" i
+        try Printf.sprintf "%i_%s" i (Hashtbl.find lhs_names i)
+        with Not_found -> Printf.sprintf "%i" i
```

Comment:
Thanks to not change this part of the code (you seem to have forgotten to update).

Reply:
Oh right